org-clubhouse.el

```diff
@@ -523,13 +546,16 @@ If the stories already have a CLUBHOUSE-ID, they are filtered and ignored."
        (when project-id
          (org-clubhouse-prompt-for-epic
            (lambda (epic-id)
-             (-map (lambda (elt)
-               (let* ((title (plist-get elt :title))
-                      (story (org-clubhouse-create-story-internal
-                            title
-                            :project-id project-id
-                            :epic-id epic-id)))
-               (org-clubhouse-populate-created-story elt story))) new-elts))))))))
+             (org-clubhouse-prompt-for-story-type
```

Comment:
definitely don't think it'd be too hard to check a config variable here, and allow defaulting to "Feature" rather than prompting every time

Reply:
How do you envision UX for switching story types? Bring up an interactive function that sets the new story type? Would user run interactive function a second time to switch back to their default story type? 

Prompting on each creation doesn't seem like a lot of overhead. That said, I'm happy to change - just need some clarification on the proposed UX